Plugins/OutText.Internal.ps1

```diff
@@ -161,6 +181,18 @@
                 ## Fix Set-StrictMode
                 if (Test-Path -Path Variable:\Options) { $options = Get-Variable -Name Options -ValueOnly; }
                 else { $options = New-PScriboTextOptions; }
+
+                try{
```

Comment:
I think you're also missing the `TextWidth` check? Can you change the `begin` script block to:
```powershell
            begin {
                ## Fix Set-StrictMode
                if (Test-Path -Path Variable:\Options) {
                    $options = Get-Variable -Name Options -ValueOnly;
                    if (-not ($options.ContainsKey('SeparatorWidth'))) {
                        $options['SeparatorWidth'] = 120;
                    }
                    if (-not ($options.ContainsKey('LineBreakSeparator'))) {
                        $options['LineBreakSeparator'] = '_';
                    }
                    if (-not ($options.ContainsKey('TextWidth'))) {
                        $options['TextWidth'] = 120;
                    }
                }
                else { $options = New-PScriboTextOptions; }
            }
```